src/jobs/SyncOktaUsersJob.php

```diff
@@ -76,16 +77,26 @@ public function getTitle()
      */
     public function process()
     {
-        // make an api call to get the list of users from okta
-        $users = Injector::inst()
-            ->get(OktaService::class)
-            ->getAllUsers(100, Config::inst()->get(SyncOktaUsersJob::class, 'statuses_to_sync'));
+        // For each of the user group filters make an api call to get the list of users from okta
+        $allGroupIDs = OktaUserGroupFilter::get();
+        $allUsers = [];
+        foreach ($allGroupIDs as $allGroupID) {
```

Comment:
This is a bit confusing

foreach (OktaUserGroupFilter::get() as $oktaUserGroupFilter) is much more obvious what's going on

Reply:
Sweet I will modify.